core/state/statedb.go

```diff
@@ -86,6 +87,7 @@ func (m *mutation) isDelete() bool {
 // commit states.
 type StateDB struct {
 	arbExtraData *ArbitrumExtraData // must be a pointer - can't be a part of StateDB allocation, otherwise its finalizer might not get called
+	arbTxFilter  int
```

Comment:
Maybe we could use an enum instead of an int so that each case could have a clear human-readable name in the code?